platform/admin/configurator/admin-configurator-actions-api/src/main/java/org/codice/ddf/internal/admin/configurator/actions/ConfiguratorSuite.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.internal.admin.configurator.actions;
+
+import org.codice.ddf.admin.configurator.ConfiguratorFactory;
+
+/** Provides access to a ConfiguratorFactory and a collection Actions */
+public interface ConfiguratorSuite {
+
+  ConfiguratorFactory getConfiguratorFactory();
```

Comment:
❓ Should we not document public methods like these?

Reply:
I figured that was covered by the interface description and the method return type.